jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/ClickAwayListenerFrame.java

```diff
@@ -0,0 +1,20 @@
+package io.github.com.pages.utils;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.material.elements.displaydata.Divider;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.tools.func.JFunc1;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class ClickAwayListenerFrame extends Section {
+    @UI("#root > div > button")
+    public UIElement openMenuButton;
```

Comment:
Need to create class ClickAwayListener and make this variable with this class

Reply:
It was decided to refrain from creating additional classes with no need in custom asserts. Now we are using TextAssert of Text class.